src/hypervisor_bootloader/src/main.rs

```diff
@@ -717,6 +828,7 @@ fn set_up_el1() {
     /* Ignore it currently... */
 
     /* CPACR_EL1 & CPTR_EL2 */
+    set_cptr_el2(0x0);
```

Comment:
Is this necessary change? This looks a destructive change and inserted in your debugging.
(Sorry I missed this before.)

Reply:
Yes, this change is necessary for U-Boot. They enables some flags, and this made exception when the boot loader accessed ZCR_EL2. I'm not sure this might happen in some UEFI environment. So, added feature flag for this sentence.